### PR TITLE
Fixing wrong check

### PR DESCRIPTION
**Pull request by frezbo**
Signed-off-by: Noel Georgi <18496730+frezbo@users.noreply.github.com>

### CODE REVIEW

**Comment by frezbo** [comment]
@moul PTAL

File: pkg/commands/proxy.go [hunk 1]
```diff
@@ -198,7 +198,7 @@ func expandSSHTokens(tokenized string, host *config.Host, gateway *config.Host)
 }
 
 func prepareHostControlPath(host, gateway *config.Host) error {
-	if !config.BoolVal(host.ControlMasterMkdir) && ("none" == host.ControlPath) {
+	if !config.BoolVal(host.ControlMasterMkdir) && ("" == host.ControlPath) {
```
Comment (moul) [inline pkg/commands/proxy.go]: Can you check both values?

`"none"` is a valid value that should have the same effect as `""`

**Comment by frezbo** [comment]
@moul fixed. Any plans for a new release. It's been quite some time.

**Comment by moul** [comment]
thank you, will do a new release soon :)

**Pull request merged**